src/applications/mhv-secure-messaging/tests/components/Compose/ComposeForm.unit.spec.jsx

```diff
@@ -920,17 +921,62 @@ describe('Compose form component', () => {
     );
     inputVaTextInput(screen.container, 'Test User', signatureTextFieldSelector);
     expect(signatureTextField).to.have.attribute('error', '');
+  });
+
+  it('displays an error in Electronic Signature component if checkbox is not checked', async () => {
+    const screen = renderWithStoreAndRouter(
+      <ComposeForm recipients={initialState.sm.recipients} />,
+      {
+        initialState,
+        reducers: reducer,
+      },
+    );
+    // Enters value for all other compose form fields first
+    const tgRecipient = initialState.sm.recipients.allowedRecipients.find(
+      r => r.signatureRequired,
+    ).id;
+    selectVaSelect(screen.container, tgRecipient);
+
+    await waitFor(() => {
+      selectVaRadio(screen.container, 'COVID');
+      expect(
+        $('va-radio-option[value="COVID"]', screen.container),
+      ).to.have.attribute('checked', 'true');
+    });
+
+    const subjectInput = 'Test Subject';
+    inputVaTextInput(screen.container, subjectInput);
+    await waitFor(() => {
+      expect(screen.getByTestId('message-subject-field')).to.have.value(
+        subjectInput,
+      );
+    });
+
+    const messageBody = 'test body';
+    inputVaTextInput(screen.container, messageBody, 'va-textarea');
+    await waitFor(() => {
+      expect(screen.getByTestId('message-body-field')).to.have.value(
+        messageBody,
+      );
+    });
 
```

Comment:
Electronic signature and checkbox errors no longer require all other fields to be populated before displaying.. Removing all of this text